### PR TITLE
IDEMPIERE-4658 The operator Dropdown entries for a foreign ID column is corrupted when a search preset is loaded in any window's Advanced Search tab.

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**
https://idempiere.atlassian.net/browse/IDEMPIERE-4658

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java [hunk 1]
```diff
@@ -1228,7 +1229,7 @@ private void setValues(Listbox listColumn, Listbox listOperator, String[] fields
             	{
                 	listColumn.setSelectedItem(li);
             		selected = true;
-            		break;
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java:1231]: why the need to remove the break ?

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java [hunk 2]
```diff
@@ -1242,10 +1243,12 @@ private void setValues(Listbox listColumn, Listbox listOperator, String[] fields
             	{
             		listOperator.setSelectedItem(li);
             		selected = true;
-            		break;
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java:1245]: same here, why remove the break ?

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java (hunk 1 above)
Comment (CarlosRuiz-globalqss) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java:1231]: The list of columns was being shown incomplete.

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java (hunk 2 above)
Comment (CarlosRuiz-globalqss) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/FindWindow.java:1245]: The list of operators was being filled incomplete.

**Review by hengsin — APPROVED** [review]
good to go

**Comment by Soundbytes** [comment]
I just pulled the patch and tested it. It works as expected. 
Thanks for the quick response. :)

**Pull request merged**